src/pages/index.tsx

```diff
@@ -40,23 +42,19 @@ const Index: FC = () => {
 
       <div className={classNames.container}>
         {!!error && <p>Something went wrong, please contact admin!</p>}
-        {
-          isLoading
-            ? (
-              <p>Loading...</p>
-            ) : (
-              <>
-                {
-                  Object.keys(filteredTask).length > 0
-                    ? Object.keys(filteredTask).map((key) => (
-                      <Accordion open title={key} key={key}>
-                        {renderCardList(filteredTask[key])}
-                      </Accordion>
-                    )) : (!error && 'No Tasks Found')
-                }
-              </>
-            )
-        }
+        {isLoading ? (
+          <p>Loading...</p>
+        ) : (
+          <>
+            {Object.keys(filteredTask).length > 0
+              ? Object.keys(filteredTask).map((key) => (
+                <Accordion open title={key} key={key}>
+                  {renderCardList(filteredTask[key])}
+                </Accordion>
+              ))
+              : !error && 'No Tasks Found'}
+          </>
+        )}
```

Comment:
Can we revert these indentation changes?

Reply:
Made changes. Please check 😁

Reply:
Can't see any changes